e2e_tests/tests/experiment/test_custom_searcher.py

```diff
@@ -31,7 +34,12 @@ def test_run_custom_searcher_experiment() -> None:
     config["description"] = "custom searcher"
     search_method = SingleSearchMethod(config)
     search_runner = SearchRunner(search_method)
-    search_runner.run(config, context_dir=conf.fixtures_path("no_op"))
+    experiment_id = search_runner.run(config, context_dir=conf.fixtures_path("no_op"))
+
+    if client._determined is not None:
```

Comment:
nitpick: I wonder if it would have worked to just do
```
assert client._determined is not None
```

Reply:
It totally would!